### DISCUSSION TITLE
Add FAQ for Specification

### DISCUSSION THREAD

**Discussion by navveenb** [Green-Software-Foundation/sci#100] [General]
Adding FAQ would be good to address frequently asked questions and some of the discussions can be turned to FAQ. Some of the discussions that we had yesterday, around proxy computation for serverless, when to choose telemetry v/s benchmark would be helpful to document as a FAQ.

**Comment by jawache** [comment]
This is a great point, I believe it should however be outside the specification, or at least not in the same file, perhaps an FAQ.md? Shall we use this thread to capture the FAQs for now?

**Reply by navveenb** [reply]
Yes, I meant a new page should be created, outside the specification. We will use this thread to capture FAQ.

**Comment by atg-abhishek** [comment]
@navveenb and @jawache yes this is a good idea! My recommendation is to use the [wiki](https://github.com/Green-Software-Foundation/software_carbon_intensity/wiki) for this rather than creating a new `.md` file that will help to achieve both the separation from the SCI documents and make it easier for managing contributions to it through a non-code route as well. 

1. The wiki has a separate clone link - https://github.com/Green-Software-Foundation/software_carbon_intensity.wiki.git
2. Easy way to add multiple pages that have a natural organization on the sidebar
![image](https://user-images.githubusercontent.com/3943612/135706745-5a595343-aa5c-4eec-aded-0ea78daf7432.png)
